io-package.json

```diff
@@ -14,8 +14,20 @@
             "pl": "ZWave wsparcie oparte na pakiecie openzwave",
             "zh-cn": "ZWave支持基于openzwave数据包"
         },
-        "version": "1.7.2",
+        "version": "1.8.0",
```

Comment:
Like we discussed earlier - calling ldconfig is a breaking change, since it requires that a recent fixer has been executed. Therefore this MUST be a major version bump (2.0.0).

Reply:
Dummy me  .. you are right of course